packages/wallet-connect-sdk-core/src/index.ts

```diff
@@ -262,33 +607,106 @@ export class WcSdk {
         } catch (error) {
             return {
                 method: request.method,
-                result: {error},
+                result: { error },
             }
         }
     }
 
-    static async invokeFunction(wcClient: Client, session: SessionTypes.Created, chainId: string, request: ContractInvocation) {
+    /**
+     * Sends an 'invokefunction' request to the Wallet and it will communicate with the blockchain. It will consume gas and persist data to the blockchain.

```

Comment:
```suggestion
     * Sends an 'invokefunction' request to the Wallet, which should approve it and communicate the transaction to the blockchain. It will consume gas and persist data to the blockchain.

```